### DISCUSSION TITLE
Auto scroll opened group to top of view

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16840] [Feature Requests]
With 10+ groups in accordion, when opening the last ones, most of their content stays out of view.
I'd like to have the group jump to the top of the view.
For instance, registering an event on the click that opens the group, in order to scroll down the `#main-content` and bring the group's content in view?

**Comment by rijkvanzanten** [comment]
I'll convert this question to a feature request to have this in :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
